readme.md

```diff
@@ -28,54 +28,87 @@ TODO:
 - [ ] create new chat
 - [ ] bots (bot keyboard)
 
+## Requirements
 
-## Installation
+To use tg, you'll need to have the following installed:
+
+- [Python 3.8](https://www.python.org/downloads/release/python-380/)
+
+## Optional dependencies
+
+- [terminal-notifier](https://github.com/julienXX/terminal-notifier) for Mac (used by default). You can change it to [dunst](https://github.com/dunst-project/dunst) for Linux or any other notifications program (see `NOTIFY_CMD` in configuration)
+- [ffmpeg](https://ffmpeg.org/) to record voice msgs and upload videos.
+- [tdlib](https://tdlib.github.io/td/build.html?language=Python) - in case of incompatibility with built in package.
+  For example, macOS:
+  ```sh
+  brew install tdlib
+  ```
+  and then set in config `TDLIB_PATH`
+- `urlview` to choose urls when there is multiple in message, use `URL_VIEW` in config file to use another app (it should accept urls in stdin)
+- to open `stickers` and `animated` ones (thumbnail preview) you need to set in mailcap appropriate handler and have app which will open `webp` file:
+  ```ini
+  image/webp; mpv %s
+  ```
 
-`python3.8` required.
+## Installation
 
+### From PyPI
 
-From pip:
+This option is recommended for production:
 
 ```sh
 pip3 install tg
+tg
 ```
 
-From sources:
+### Using flit
+
+This option is recommended for development:
+
+> Requires [flit](https://github.com/takluyver/flit) to be installed.
+>
+> Install it with:
+> ```sh 
+> pip3 install flit
+> ```
 
 ```sh
-pip3 install python-telegram
-git clone git@github.com:paul-nameless/tg.git
+git clone https://github.com:paul-nameless/tg.git
 cd tg
-PYTHONPATH=. python3 tg/main.py
+flit install
+tg
 ```
 
-Docker (voice recordings and notifications won't work):
+### Without installing
+
+> Ensure you have the correct version of Python installed before using this method!
+
 ```sh
-docker run -it --rm tg
+git clone https://github.com:paul-nameless/tg.git
+cd tg
+python3 -m venv venv
+source venv/bin/activate
+pip install python-telegram
+python3 tg/__main__.py
```

Comment:
why aren't you using `python3 -m tg` here? I think idea of this PR was so allow using such way of running a module :slightly_smiling_face:

Reply:
Well, it isn't installed as a module, so using `-m` won't work.

However, now that I think of it, this should work:
```suggestion
python3 tg
```